contributing/releasing.md

```diff
@@ -235,6 +224,19 @@ go-ahead from the following clients:
 
 ---
 
+#### Verify CocoaPods podspec and trunk access
+
+Send our local podspec through the CocoaPods linter:
+
+    pod lib lint MaterialComponents.podspec --skip-tests
+
+CocoaPods publishes a directory of publicly available pods through its **trunk** service.
+Note: Ensure that you can [push the podspec](#publish-to-cocoapods) later by checking for `MaterialComponents` in your list of available `Pods` when you:
+
+    pod trunk me
+
+If this fails or MaterialComponents is not listed [register an account and session](https://guides.cocoapods.org/making/getting-setup-with-trunk.html).
```

Comment:
Can this remain where it was? I thought this step was to verify that the release engineer's permissions are correct. In that case, doing it earlier is better than later (so they can fix the issues while waiting on internal testing/approval).